src/converter.js

```diff
@@ -259,9 +259,11 @@ var getHeadersArray = function(swagger, path, method) {
     // Need to check OAS 3.0 spec about type http and scheme
     for (var m in swagger.security) {
       var secScheme = Object.keys(swagger.security[m])[0]
-      var secDefinition = swagger.components.securitySchemes[secScheme]
-      var authType = secDefinition.type.toLowerCase()
-      let authScheme = secDefinition.scheme.toLowerCase()
+      const { securitySchemes, securityDefinitions } = swagger.components ? swagger.components : swagger;
+      const secDefinition =  securitySchemes ? securitySchemes[secScheme] : securityDefinitions[secScheme];
+      const { type, scheme } = secDefinition;
```

Comment:
If spec doesn't define `security` schemes at all, you will get `TypeError: Cannot destructure property ...`